.github/workflows/node.js.yml

```diff
@@ -28,3 +28,6 @@ jobs:
     - run: npm install
     - run: npm run build
     - run: npm run lint
+    - uses: JS-DevTools/npm-publish@v1
```

Comment:
We want to do this in a separate workflow file which only runs on push.

Reply:
I think I did this correctly now. I am not completely sure how to test it.

Reply:
I don't think there's a good way to test. We just need to land it, bump the version and see what happens. Have you set up the NPM token secret?